subway_prediction_ml/actuals_adder.py

```diff
@@ -51,4 +62,23 @@ def _load_actuals(self):
         raw_frame['stop_id'] = raw_frame['stop_id'].astype('str')
         dropped_frame = raw_frame.dropna(subset=["time"])
         is_subway = dropped_frame["trip_id"].apply(lambda x: x[0:3] != "CR-")
-        return dropped_frame[is_subway].drop_duplicates()
+        subway_actuals = dropped_frame[is_subway].drop_duplicates()
+        subway_actuals = subway_actuals.rename(
+            {"trip_id": "gtfs_trip_id", "stop_id": "destination_gtfs_id"},
+            axis="columns"
+        )
+        subway_actuals["gtfs_trip_id"] = \
+            subway_actuals["gtfs_trip_id"].astype(str)
+        return subway_actuals
+
+    def _service_date_for_timestamp(self, timestamp):
```

Comment:
It seems like the timestamp column is parsed from a string to a datetime, then to a timestamp, then back to a datetime. Maybe use a `datetime.timedelta` to modify the original datetime directly?

Reply:
@losvedir even though it means an extra step or two in the conversion, I think there's value in having both `timestamp` and `time` obviously pass through the same logic, as embodied by this method.